packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1662,6 +1662,122 @@ describe("IModelTransformerHub", () => {
     sinon.restore();
   });
 
+  it("should be able to handle a transformation which deletes a relationship and then elements of that relationship", async () => {
+    const masterIModelName = "Master";
```

Comment:
in case of collisions can you rename this to `MasterDeleteRelAndEnds`?

Reply:
done